src/main/java/com/spartronics4915/frc2020/commands/LowerPanelRotatorCommand.java

```diff
@@ -0,0 +1,39 @@
+package com.spartronics4915.frc2020.commands;
```

Comment:
Keep them for now, but we'll probably end up making `Raise` and `Lower` "inline commands". Notice the simplicity of these two - all you're doing is calling one of `mPanelRotator`'s methods in the `execute`, `isFinished`, and `end` loops... that's only a little bit of code for making a completely new file.

WPILib lets you make inline commands by using lambdas and method references - essentially constructing a nameless class. You can pass through "method references" (note: _not_ calling a method) to one of WPI's special Command classes, letting you cut down this entire file to two lines of code. If you're interested in reading up on it (highly recommend!), [the documentation is available here.](https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html)